src/kvstore/KVStore.h

```diff
@@ -192,8 +192,8 @@ class KVStore {
 
     virtual ResultCode flush(GraphSpaceID spaceId) = 0;
 
-    virtual ErrorOr<ResultCode, std::string> createCheckpoint(GraphSpaceID spaceId,
-                                                              const std::string& name) = 0;
+    virtual ErrorOr<ResultCode, std::pair<std::string, nebula::cpp2::PartitionBackupInfo>>
```

Comment:
We need to change the data structure of resp. If a space is mounted on multiple disks, only one path is not sufficient.
Something like `map<path, PartitionBackupInfo>` is ok.

Reply:
Talked offline, will fix in #377